tests/FakeItEasy.IntegrationTests/ExceptionMessagesTests.cs

```diff
@@ -35,13 +35,13 @@ public void Should_give_pretty_message_when_trying_to_fake_extension_method()
             var fake = A.Fake<List<int>>();
 
             // Act
-            var exception = Record.Exception(() => A.CallTo(() => fake.Any()));
+            var exception = Record.Exception(() => A.CallTo(() => fake.Average()));
 
             // Assert
             var expectedMessage =
 @"
 
-  The current proxy generator can not intercept the method System.Linq.Enumerable.Any`1[System.Int32](System.Collections.Generic.IEnumerable`1[System.Int32] source) for the following reason:
+  The current proxy generator can not intercept the method System.Linq.Enumerable.Average(System.Collections.Generic.IEnumerable`1[System.Int32] source) for the following reason:

```

Comment:
There was nothing special about `Any` (nor is there about `Average`). Any extension method will do.